api/MediaDevices.json

```diff
@@ -208,40 +211,65 @@
+              "version_added": "53",
+              "notes": [
+                "Older versions of Chrome and Opera implement navigator.webkitGetUserMedia, the prefixed version of the legacy navigator.getUserMedia method. From version 47 to 52, the promise-based interface was only available through the adapter.js polyfill, or using the flag chrome://flags/#enable-experimental-web-platform-features. Starting with version 53, the promise-based interface is on by default, though that interface is still not available through navigator.",
+                "Chrome throws error if the page serving the script is loaded from insecure origin (i.e. HTTP)."
+              ]
             },
             "chrome": {
-              "version_added": null
+              "version_added": "53",
+              "notes": [
+                "Older versions of Chrome and Opera implement navigator.webkitGetUserMedia, the prefixed version of the legacy navigator.getUserMedia method. From version 47 to 52, the promise-based interface was only available through the adapter.js polyfill, or using the flag chrome://flags/#enable-experimental-web-platform-features. Starting with version 53, the promise-based interface is on by default, though that interface is still not available through navigator.",
+                "Chrome throws error if the page serving the script is loaded from insecure origin (i.e. HTTP)."
+              ]
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": "53",
+              "notes": [
+                "Older versions of Chrome and Opera implement navigator.webkitGetUserMedia, the prefixed version of the legacy navigator.getUserMedia method. From version 47 to 52, the promise-based interface was only available through the adapter.js polyfill, or using the flag chrome://flags/#enable-experimental-web-platform-features. Starting with version 53, the promise-based interface is on by default, though that interface is still not available through navigator.",
+                "Chrome throws error if the page serving the script is loaded from insecure origin (i.e. HTTP)."
+              ]
             },
             "edge": {
-              "version_added": null
+              "version_added": true
             },
             "edge_mobile": {
-              "version_added": null
+              "version_added": true
             },
             "firefox": {
-              "version_added": null
+              "version_added": "36",
+              "notes": [
+                "Older versions of Firefox implement navigator.mozGetUserMedia(), the prefixed version of the legacy navigator.getUserMedia method.",
+                "Prior to Firefox 55, getUserMedia() incorrectly returns NotSupportedError when the list of constraints specified is empty, or has all constraints set to false. Starting in Firefox 55, this situation now correctly calls the failure handler with a TypeError."
```

Comment:
This note could use a few adjustments:

`"Before Firefox 55, <code>getUserMedia()</code> incorrectly returns <code>NotSupportedError</code> when the list of constraints specified is empty, or has all constraints set to <code>false</code>. Starting in Firefox 55, this situation now correctly calls the failure handler with a <code>TypeError</code>."`

Reply:
Agree